app/models/resource.rb

```diff
@@ -1,6 +1,6 @@
 class Resource < ApplicationRecord
   include Elasticsearch::Model
-  index_name "articles-#{Rails.env}"
+  index_name "#{self.class.name.downcase}-#{Rails.env}"
```

Comment:
Oh ok, didn't know these characters were not allowed. At least, now I know we're both into slashes! And good call on extracting it, going to be super easy to reuse like this.